packages/amplify-cli/src/plugin-helpers/platform-health-check.ts

```diff
@@ -91,21 +91,27 @@ function isMatching(pluginDescription: PluginDescription, pluginInfo: PluginInfo
   return result;
 }
 
-export function getOfficialPlugins() {
+export function getOfficialPlugins(): { [key: string]: PluginDescription | Array<PluginDescription> } {
   const packageJsonFilePath = path.normalize(path.join(__dirname, '../../package.json'));
   const packageJson = JSONUtilities.readJson<$TSAny>(packageJsonFilePath);
   const { officialPlugins } = packageJson.amplify;
 
   const dependencies: { [key: string]: string } = packageJson.dependencies;
 
   Object.keys(officialPlugins).forEach((plugin: string) => {
-    const { packageName } = officialPlugins[plugin];
-    if (dependencies[packageName]) {
-      const version = dependencies[packageName];
-      officialPlugins[plugin].packageVersion = version;
-    } else {
-      delete officialPlugins[plugin].packageVersion;
+    let plugins = officialPlugins[plugin];
```

Comment:
nit: consider using `const`
const plugins = Array.isArray(officialPlugins[plugin]) ? officialPlugins[plugin] : [officialPlugins[plugin]];

Reply:
Thank you @lazpavel, using `const` looks good.